jsx/Tabs.js

```diff
@@ -171,7 +171,62 @@ TabPane.propTypes = {
   activeTab: React.PropTypes.string
 };
 
+class Modal extends React.Component {
+
+  render() {
+    // Render nothing if the "show" prop is false
+    if (!this.props.show) {
+      return null;
+    }
+
+    // White Modal Window
+    const modalStyle = {
```

Comment:
i agree with @driusan in that the bootstrap modal component could be used in place of this. however, if we are looking to add this feature as LORIS specific, i think we can put it into a more robust package where the css is external to this file and is customizable